### PR TITLE
Release-6.4.0

### PR DESCRIPTION
**Pull request by AdamL-Microsoft**
### Added 
* Deployment/CLI: OneFuzz Config refactored - `tenant_id`, `tenant_domain`, `multi_tenant_domain`, and `cli_client_id` are now required values in the config.json used during deployment and no longer required when running the config command. [#2771](https://github.com/microsoft/onefuzz/pull/2771), [#2811](https://github.com/microsoft/onefuzz/pull/2811)
* Agent: Fully escape allowlist rules [#2784](https://github.com/microsoft/onefuzz/pull/2784)
* Agent: Apply allowlist to all blocks within a function [#2785](https://github.com/microsoft/onefuzz/pull/2785)
* CLI: Added a cli subcommand `onefuzz debug notification template` to validate scriban notification templates [#2800](https://github.com/microsoft/onefuzz/pull/2800)
* Service: Added Notification failure webhook to communicate Notification failures [#2628](https://github.com/microsoft/onefuzz/pull/2628)
### Changed
* Service: Include `AssignedTo` when failing to create a work item due to an authentication exception [#2770](https://github.com/microsoft/onefuzz/pull/2770)

### Fixed
* Agent: Fixes & improvements to `Expand` behavior [#2789](https://github.com/microsoft/onefuzz/pull/2789)
* Agent: Triming whitespace in output from monitored process before printing [#2782](https://github.com/microsoft/onefuzz/pull/2782)
* CLI: Fixed default value of analyzer_exe [#2797](https://github.com/microsoft/onefuzz/pull/2797)
* CLI: Fixed missing `readonly_inputs` parameter in dotnet & dotnet_dll templates [#2740](https://github.com/microsoft/onefuzz/pull/2740)
* Service: Fixed query to get the existing proxy [#2791](https://github.com/microsoft/onefuzz/pull/2791)
* Service: Truncate webhooks message length for azure queue size compatibility [#2788](https://github.com/microsoft/onefuzz/pull/2788)

### CODE REVIEW

**Review by mgreisen — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 6.4.0
+
+### Added 
+* Deployment/CLI: OneFuzz Config refactored - `authority`, `tenant_domain`, and `cli_client_id` are now required values in the config.json used during deployment [#2771](https://github.com/microsoft/onefuzz/pull/2771)
```
Comment (nharper285) [inline CHANGELOG.md:10]:
```suggestion
* Deployment/CLI: OneFuzz Config refactored - `tenant_id`, `tenant_domain`, `multi_tenant_domain`, and `cli_client_id` are now required values in the config.json used during deployment and no longer required at when running the config command. [#2771](https://github.com/microsoft/onefuzz/pull/2771)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
...
+### Changed
+* Service: Include `AssignedTo` when failing to create a work item due to an authentication exception [#2770](https://github.com/microsoft/onefuzz/pull/2770)
+
+### Fixed
+* Agent: Fixes & improvements to `Expand` behavior [#2789](https://github.com/microsoft/onefuzz/pull/2789)
+* Agent: Triming whitespace in output from monitored process before printing [#2782](https://github.com/microsoft/onefuzz/pull/2782)
+* CLI: Fixed default value of analyzer_exe [#2797](https://github.com/microsoft/onefuzz/pull/2797)
+* Service: Fixed query to get the existing proxy [#2791](https://github.com/microsoft/onefuzz/pull/2791)
+* Service: Truncate webhooks message length for azure queue size compatibility [#2788](https://github.com/microsoft/onefuzz/pull/2788)
+
```
Comment (nharper285) [inline CHANGELOG.md:23]: Were you able to include the Notification failed webhook PR? If so, can you add it to the changelog?

**Review by nharper285 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mgreisen) [reply CHANGELOG.md:10]: You might want to remove the `at` in `no longer required at`

Comment (nharper285) [reply CHANGELOG.md:10]: updated.

File: CHANGELOG.md (hunk 2 above)
Comment (AdamL-Microsoft) [reply CHANGELOG.md:23]: Yes the PR is included in the release, but missed adding it to the changelog. Added now.

**Review by nharper285 — APPROVED** [review]

**Pull request merged**